web/models.py

```
@@ -678,6 +678,24 @@ class EducationalVideo(models.Model):
         help_text="User who uploaded the video. If null, the submission is considered anonymous.",
     )
     uploaded_at = models.DateTimeField(auto_now_add=True)
+    video_id = models.CharField(
+        max_length=12, unique=True, editable=False, default="", help_text="Auto-generated unique video identifier."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)